site/src/api/queries/groups.ts

```diff
@@ -6,18 +6,18 @@ import type {
   PatchGroupRequest,
 } from "api/typesGenerated";
 
-const GROUPS_QUERY_KEY = ["groups"];
 type GroupSortOrder = "asc" | "desc";
 
 const getGroupQueryKey = (organizationId: string, groupName: string) => [
+  "organization",
   organizationId,
   "group",
   groupName,
 ];
 
 export const groups = (organizationId: string) => {
   return {
-    queryKey: GROUPS_QUERY_KEY,
+    queryKey: ["organization", organizationId, "groups"],
```

Comment:
I see that this query key is being used in multiple places. Should we centralize it to ensure consistency?